src/main/scala/net/snowflake/spark/snowflake/CloudCredentialsUtils.scala

```diff
@@ -19,26 +19,31 @@ package net.snowflake.spark.snowflake
 
 import java.net.URI
 
-import com.amazonaws.auth.{BasicAWSCredentials, AWSCredentials, AWSSessionCredentials, InstanceProfileCredentialsProvider}
+import com.amazonaws.auth.{AWSCredentials, AWSSessionCredentials, BasicAWSCredentials, InstanceProfileCredentialsProvider}
+import com.microsoft.azure.storage.StorageCredentialsSharedAccessSignature
 import net.snowflake.spark.snowflake.Parameters.MergedParameters
 import org.apache.hadoop.conf.Configuration
 import org.apache.spark.sql.SQLContext
 
-private[snowflake] object AWSCredentialsUtils {
+private[snowflake] object CloudCredentialsUtils {
   /**
    * Generates a credentials string for use in Snowflake COPY in-out statements.
    */
   def getSnowflakeCredentialsString(sqlContext: SQLContext,
-                                    params: MergedParameters): String = {
+                                          params: MergedParameters): String = {
```

Comment:
Minor nitpick: Would it be possible to preserve original alignment? :)

Reply:
Changed.